koku/masu/processor/parquet/parquet_report_processor.py

```diff
@@ -187,9 +188,9 @@ def create_table(self):
     def file_extension(self):
         """File format compression."""
         first_file = self.file_list[0]
-        if first_file.lower().endswith(CSV_EXT):
+        if first_file.name.endswith(CSV_EXT):
```

Comment:
This probably still needs the `lower()`. You could also use `first_file.suffix == CSV_EXT` but that won't handle compound suffixes like `csv.gz`.